metecho/api/models.py

```diff
@@ -1198,7 +1206,7 @@ class ScratchOrg(
     description = MarkdownField(blank=True, property_suffix="_markdown")
     org_type = StringField(choices=ScratchOrgType.choices)
     org_config_name = StringField()
-    owner = models.ForeignKey(User, on_delete=models.PROTECT)
+    owner = models.ForeignKey(User, on_delete=models.PROTECT, blank=True, null=True)
```

Comment:
This requires changes to `orgs/reducer.ts` where we define the Org `owner: string` (should now be `owner: string | null`).